test/integration/synthetic_study_populator_test.rb

```diff
@@ -24,16 +29,16 @@ class SyntheticStudyPopulatorTest < ActionDispatch::IntegrationTest
   teardown do
     # this will also destroy the dependent annotation and assembly
     @taxon.destroy!
+    @study = Study.find_by(name: SYNTH_STUDY_INFO[:name])
```

Comment:
```suggestion
    study = Study.find_by(name: SYNTH_STUDY_INFO[:name])
```
I don't think this needs to be an instance variable

Reply:
Agreed